docs/designers-developers/developers/filters/editor-filters.md

```diff
@@ -30,3 +29,24 @@ var customPreviewMessage = function() {
 wp.hooks.addFilter( 'editor.PostPreview.interstitialMarkup', 'my-plugin/custom-preview-message', customPreviewMessage );
 ```
 
+## Editor settings
+
+### `block_editor_settings`
+This is a PHP filter which is applied before sending settings to the Editor in `wp-admin/edit-form-blocks.php`.
+
+You may find details about this filter [on its WordPress Code Reference page](wp-admin/edit-form-blocks.php).
```

Comment:
This link is broken. It needs to be the full URL.

Reply:
ugh, bad copy paste apparently